### ISSUE TITLE
Issue on page /svm_decoding.html

### ISSUE THREAD

**Issue by Andjelaaaa** [main-educational/brain_encoding_decoding#37]
Hi, at the last step where the accuracy is printed for the Fast Regularized Ensembles of models (frem) as follows:
`print('F1 scoreswith FREM')
for category in categories:
    print(category, '\t\t    {:.2f}'.format(np.mean(decoder.cv_scores_[category])))`
Shouldn't it be averaged using frem scores instead of the ones from the decoder model? Hence, it would be:
`print('F1 scoreswith FREM')
for category in categories:
    print(category, '\t\t    {:.2f}'.format(np.mean(frem.cv_scores_[category])))`

**Comment by PeerHerholz** [comment]
Hi @Andjelaaaa,

thank you very much for opening this issue and pointing this out.

I think you are correct. Just checking with @bthirion who prepared and presented this part: that's a typo, right?

Thx again.

Cheers, Peer

**Comment by bthirion** [comment]
You're absolutely right ! Sorry for the mistake

**Comment by PeerHerholz** [comment]
Thx for the reply @bthirion and no worries at all!

@Andjelaaaa would you be up for submitting a small PR to fix this? Otherwise, I can do it as well.

**Comment by pbellec** [comment]
was that fixed in the last update @PeerHerholz ?

**Comment by PeerHerholz** [comment]
Yes, this was updated and fixed in the [respective section](https://main-educational.github.io/brain_encoding_decoding/svm_decoding.html#getting-more-meaningful-weight-maps-with-frem) via #43 .
